packages/server/src/fhir/structure.test.ts

```diff
@@ -0,0 +1,18 @@
+import { getSearchParameter } from "@medplum/core";
+import { loadStructureDefinitions } from "./structure";
+
+describe('FHIR structure', () => {
+    beforeAll(async () => {
+      loadStructureDefinitions();
+    });
+
+    test('Can search Organization on _profile', async () =>{
+        const param = getSearchParameter('Organization', '_profile');
+        expect(param).toBeDefined();
+    });
+
+    test('Can search MedicinalProductManufactured on _profile', async () =>{
+        const param = getSearchParameter('MedicinalProductManufactured', '_profile');
+        expect(param).toBeDefined();
+    });
+});
```

Comment:
Looks like this file is missing a newline at the end
```suggestion
});

```

Reply:
Seems to be resolved by autofix.ci